### PR TITLE
BLOCKED until 2.0 is released - Feature - Export scenario

### PR DESCRIPTION
**Pull request by khaitruong922**

### CODE REVIEW

File: docs/manuals/core/entities/scenario-cycle-mgt.md [hunk 1]
```diff
@@ -135,6 +135,19 @@ You can define a list of authorized tags in the scenario configuration by specif
...
+
+You can export a scenario with its related entities into a folder using `taipy.export_scenario()^`. This method takes the scenario id and the path to the folder as parameters. The folder will be created if it does not exist and overwrtiten if it exists.
+
+```python linenums="1"
+import taipy as tp
+import my_config
+
+scenario = tp.create_scenario(my_config.monthly_scenario_cfg)
+tp.export(scenario.id, "./my_scenario")
+```
```
Comment (jrobinAV) [inline docs/manuals/core/entities/scenario-cycle-mgt.md]:
```suggestion
```python linenums="1"
import taipy as tp
import my_scenario_config

scenario = tp.create_scenario(my_scenario_config)
tp.export(scenario.id, "./my_scenario")
```
```

**Review by jrobinAV — APPROVED** [review]

File: docs/manuals/core/entities/scenario-cycle-mgt.md [hunk 2]
```diff
@@ -135,6 +135,19 @@ You can define a list of authorized tags in the scenario configuration by specif
 From the scenarios that are created from that configuration, if you add a tag that is not authorized, an exception will
 be raised.
 
+# Export a scenario
```
Comment (FabienLelaquais) [inline docs/manuals/core/entities/scenario-cycle-mgt.md:138]: Just make sure section titles are homogeneous.
I can see 'Cycle attributes' below.
This then should be 'Scenario exports'.

Often something like 'Exporting scenarios' is used...

**Review by FabienLelaquais — APPROVED** [review]

File: docs/manuals/core/entities/scenario-cycle-mgt.md (hunk 2 above)
Comment (khaitruong922) [reply docs/manuals/core/entities/scenario-cycle-mgt.md:138]: This is to make it consistent with headers above: Delete a scenario, Tag/untag a scenario, etc.

**Review by jrobinAV — APPROVED** [review]
Good catch Giang. 
Let s wait for the doc release before merging it, but it is good for me.

**Pull request merged**